content/getting-started.html

```diff
@@ -0,0 +1,47 @@
+---
+title: Getting Started
+layout: page
+permalink: /getting-started/
+class: page-short
+---
+
+{% capture getting_started_content %}
+{:.cwl-red}
+## Resources for Getting Started
+Below are some useful resources to getting started.  Additionally, help is available through [community resources]({{ site.url }}/#community) such as the [CWL Gitter channel](https://gitter.im/common-workflow-language/common-workflow-language).
+
+* Learn to write CWL using the [CWL user guide]({{ site.url }}/user_guide/) which provides a gentle introduction to creating CWL command line tool and workflow descriptions.
+* Browse [CWL Implementations]({{ site.url }}/#Implementations) to find a software package that's right for you.
+* Follow [CWL Recommended Practices](http://www.commonwl.org/user_guide/rec-practices/) to make your CWL tools and workflows more efficent, reusable and transparent.
+* Watch <a href="https://github.com/pitagora-galaxy/cwl/wiki/CWL-Start-Guide-JP" lang="ja" hreflang="ja">CWLの日本語での解説ドキュメント</a>  a 15 minute introduction to the CWL project in Japanese.
+* Watch a series of [video lessons about CWL](https://stepik.org/lesson/35918/step/1?unit=15070) in Russian as part of the <a href="https://stepik.org/course/1612/syllabus" lang="ru">Управление вычислениями</a> (Computation Management) free online course.
+* Find [CWL descriptions for life-science tools](https://github.com/common-workflow-library/bio-cwl-tools) to use directly or incoporate into your workflows.
+
+{:.cwl-red}
+## Background 
+### Why Use Workfows?
+
+Workflows and using workflow tools help make your computation methods portable, maintainable, reproducible and shareable. Often analysis methods contain many steps, perhaps in multiple languages and workflows help researchers document, run and maintain these complicated analyses.
+Using a workflow system rather than a series of scripts has a number of benefits including abstraction, scaling, automation, and provenance (ASAP).
```

Comment:
```suggestion
Using a workflow system rather than a series of scripts has a number of benefits including abstraction, scaling, automation, and provenance (ASAP).
<br>
```